clients/drcachesim/tracer/instr_counter.cpp

```diff
@@ -116,7 +117,8 @@ hit_instr_count_threshold(app_pc next_pc)
     }
 #endif
     dr_mutex_lock(mutex);
-    if (tracing_mode.load(std::memory_order_acquire) == BBDUP_MODE_TRACE) {
+    if (tracing_mode.load(std::memory_order_acquire) == BBDUP_MODE_TRACE ||
+        tracing_mode.load(std::memory_order_acquire) == BBDUP_MODE_L0_FILTER) {
```

Comment:
Let's avoid two repeated load-acquires (maybe the compiler will elide the 2nd: not sure though).  Given the many instances of this pattern of checking both values, let's add a helper function with one load-acquire for this check.

Reply:
Done.